service/controller/clusterapi/v31/resource/tccp/testdata/case-0-basic-test.golden

```diff
@@ -3,6 +3,10 @@ Description: Tenant Cluster Control Plane Cloud Formation Stack.
 Outputs:
   DockerVolumeResourceName:
     Value: DockerVolume8Y5CK78968
+  IngressInsecureTargetGroupsID:
+    Value: !Ref IngressInsecureTargetGroup
+  IngressSecureTargetGroupsID:
+    Value: !Ref IngressSecureTargetGroup
```

Comment:
Is it absolutely necessary to put these information into the stack outputs? We also have another approach where we have a dedicated resource to fetch information from the AWS API separately. Reason for this in several cases is to have information accessible even during upgrades. Not sure which approach would be best here.